packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx

```diff
@@ -11,8 +11,8 @@ export interface AccordionProps extends React.HTMLProps<HTMLDListElement> {
   className?: string;
   /** Adds accessible text to the Accordion */
   'aria-label'?: string;
-  /** the heading level to use */
-  headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  /** the heading level to use. Setting this to null will use definition list instead of div */
+  headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | null;
```

Comment:
how about instead of overloading this prop, add a new prop `asDefinitionList`

Reply:
Done!